.typos.toml

```diff
@@ -48,6 +48,15 @@ extend-ignore-identifiers-re = [
     "ND_ROUTER_.*",
     "ND_NEIGHBOR_.*",
     ".*_ND_option.*",
+    "bck", # Used with same length as `fwd`
+    "pn", # Use for `PoolNode` variables
+    "ffrom_[ip|port|mac]", # Used in netcontrol.
+    "complte_flag", # Existing use in exported record in base.
```

Comment:
Also wonder if the concrete identifiers should go into extend-identifiers below.

Reply:
> We should probably fix this one.

It is in a public interface so it might need a proper deprecation, https://github.com/zeek/zeek/blob/0ebcd9608f87368dc1e02d308199ce802b28b487/scripts/base/init-bare.zeek#L4250

> Also wonder if the concrete identifiers should go into extend-identifiers below.

Looking at the [`typos` docs](https://github.com/crate-ci/typos/blob/master/docs/reference.md) again I think our use of `default.extend-*` sections is incorrect they since are intended to configure replacements (but for us original and replacement are always identical); the `default.ignore-*` sections on the other hand are intended for always valid tokens.